tajo-catalog/tajo-catalog-common/src/main/proto/CatalogProtos.proto

```diff
@@ -262,6 +262,18 @@ message PartitionIdentifierProto {
   optional string partitionName = 3;
 }
 
+message PartitionsByAlgebraProto {
+  required string databaseName = 1;
+  required string tableName = 2;
+  required string algebra = 3; /** json object which contains algebra expressions */
+}
+
+message PartitionsByFilterProto {
+  required string databaseName = 1;
+  required string tableName = 2;
+  required string filter = 3; /** filter condition, for example col1 = 'TAJO' and col2 = 100. */
```

Comment:
Please use ```//``` instead of ```/**/```. 
In addition, which format is used for filter serialization?